index.js

```diff
@@ -1,44 +1,58 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log("%cQuestion 1", "color: red") 
+var counter = 1;
+var counter = 2;
+console.log(counter) 
```

Comment:
Remember we want to get away from using `var`! If you know you're going to reassign your variable, it's better to use `let`. Then you could do something like...

```javascript
let counter = 1
counter = 2
```

Best to get in the habit now and avoid using var!